web/locales/ca/messages.ftl

```diff
@@ -14,6 +14,9 @@ email-opt-in-info-sub-with-challenge = Rebreu correus com ara recordatoris d'obj
 email-opt-in-privacy-v2 = En optar per rebre correus electrònics, accepteu que Mozilla gestioni aquesta informació en els termes descrits en la seva <privacyLink>política de privadesa</privacyLink>.
 indicates-required = * Indica un camp necessari
 not-available-abbreviation = N/D
+banner-error-slow-1 = Ens sap greu, el Common Voice està tardant a carregar-se, gràcies per l'interès.
+banner-error-slow-2 = La plataforma va molt carregada últimament, estem investigant els problemes i esperem que torne prompte.
+banner-error-slow-link = Estat del projecte
 
```

Comment:
Anything in a locale other than English is going to get overwritten next time there's a sync - I don't think that's an issue with this specific deploy but just a heads up